apps/src/blockTooltips/tooltipUtils.js

```diff
@@ -0,0 +1,130 @@
+/* global ace */
+'use strict';
+
+/**
+ * @typedef {Object} parameterSlotInfo
+ * @property {string} funcName
+ * @property {number} currentParameterIndex
+ */
+
+var ONLY_WHITESPACE_REGEXP = /^[ ]*$/;
+var ENDING_OF_BLOCK_COMMENT = /^\*\/$/;
+
+function countNumberOfCharacter(string, character) {
+  return string.split(character).length - 1;
+}
```

Comment:
Neat way of doing this. Maybe a short comment on top saying this returns the number of instances of character within string.